doc/en/user/source/production/java.rst

```diff
@@ -3,98 +3,125 @@
 Java Considerations
 ===================
 
-Use supported JRE
------------------
+Use a supported Java version 
+----------------------------
 
-GeoServer's speed depends a lot on the chosen Java Runtime Environment (JRE). The latest versions of GeoServer are tested with both Oracle JRE and OpenJDK. Implementations other than those tested may work correctly, but are generally not recommended.
+GeoServer's speed depends a lot on the chosen Java Runtime Environment (JRE). The latest versions of GeoServer are tested with both OpenJDK and Oracle. Implementations other than those tested may work correctly, but are generally not recommended.
 
-Tested:
+GeoServer is built and tested with Java 8 (Recommended), `supported up to 2026 <https://adoptopenjdk.net/support.html#roadmap>`_, and Java 11, `supported up to 2024 <https://adoptopenjdk.net/support.html#roadmap>`_. We recommend these long-term-support releases for production. Java has adopted development cycle with a release every six months containing experimental features. These releases are not tested with GeoServer and are not recommended for production.
 
-* Java 11 and Java 8 - GeoServer 2.15.x and above (OpenJDK tested)
-* Java 8 - GeoServer 2.9.x and above (OpenJDK and Oracle JRE tested)
-* Java 7 - GeoServer 2.6.x to GeoServer 2.8.x (OpenJDK and Oracle JRE tested)
-* Java 6 - GeoServer 2.3.x to GeoServer 2.5.x (Oracle JRE tested)
-* Java 5 - GeoServer 2.2.x and earlier (Sun JRE tested)
+For best performance we recommend the use *OpenJDK 8* (also known as JRE 1.8).
 
-For best performance we recommend the use *Oracle JRE 8* (also known as JRE 1.8).
+.. list-table:: Title
+   :widths: 20, 10, 10, 10, 50
+   :header-rows: 1
 
-.. Further speed improvements can be released using `Marlin renderer <https://github.com/bourgesl/marlin-renderer>`__ alternate renderer.
+   * - Java Version
+     - OpenJDK
+     - Oracle
+     - Sun
+     - GeoServer Compatibility
+   * - **Java 11**
+     - tested
+     - tested
+     -
+     - GeoServer 2.15.x and above
+   * - **Java 8**
+     - tested
+     - tested
+     -
+     - GeoServer 2.9.x and above
+   * - Java 7 
+     - tested
+     - tested
+     -
+     - GeoServer 2.6.x to 2.8.x
+   * - Java 6 
+     - 
+     - tested
+     -
+     - GeoServer 2.3.x to 2.5.x
+   * - Java 5 
+     - 
+     - 
+     - tested
+     - GeoServer 2.2.x and earlier
```

Comment:
While I do not think anyone is working on Java 5 anymore this is much easier to read as a table